content/main.js

```diff
@@ -2,12 +2,12 @@ export const attributes = {
   hero: {
     firstButton: {
       icon: 'briefcase-outline',
-      link: '/#portofolio',
-      text: 'My work',
+      link: '/#portfolio',
+      text: 'Portfolio',
```

Comment:
Changed to `portfolio` so that we use the same string in the Playwright test
```
  await page.getByRole('link', { name: 'Portfolio' }).click();
  await page.waitForURL('/#portfolio');
  await expect(page.getByTestId('portfolio')).toBeInViewport();
  ```